objects.js

```diff
@@ -8520,6 +8523,48 @@ WatcherMorph.prototype.userMenu = function () {
     return menu;
 };
 
+WatcherMorph.prototype.valueExporter = function (format) {
+    var myself = this,
+        value = this.currentValue,
+        contents,
+        format = format || 'plain',
+        ide = myself.parentThatIsA(IDE_Morph);
+
+    return function () {
+        switch (format) {
+            case 'plain':
+                contents = value instanceof List ? 
+                    value.asArray().join(', ') : 
```

Comment:
> Which one is easier to deal with depends again on your use case. Anyone can search and replace commas, but not everyone knows how to %s/\n/;/g.

My primary reason for a newline is this: Both Snap! and Scratch (and BYOB 3) make working with list files separated by newlines easy. Many programing languages have a `readLine(s)` function as part of the standard library. 

Plus, it's much easier to read a long list of words separated by new lines rather than by a comma. (Well, I suppose that's open for debate - but in my experience with the stuff students use and that most people also pretty-print JSON for reading, I don't think it's that controversial.)

As far as some lists representing a single record - you're totally right that it's possible, but in my experience the output matters less in this case. If you really want a comma separated list, as a CSV then it seems easier to make a 1D list a 2D list (all you need is a wrapper `list` block) than it would be to turn the existing list into something separated by newlines -- which is currently a rather annoying process and requires knowledge of how to enter `\n` into Snap! which isn't very well known. 

Sorry for pushing this :( - Dealing with lists in a sane way has been a frequent comment for years, and all my experience suggests newlines are much easier for most students.